netlify.toml

```diff
@@ -0,0 +1,29 @@
+[Settings]
+  ID = "react-showtime"
+
+[build]
+  base = "example/"
+  publish = "build/"
+  command = "yarn run build"
```

Comment:
This is the only command required to build the site, so I decided to reserve `cibuild` for the `react-showtime` package itself (if we even use it at all).